doc/features.md

```diff
@@ -0,0 +1,75 @@
+Network Module Features
+-----------------------
+
+Network Services
+======================
+
+- wicked
+- Network Manager
+- no network service
+- still support old netconfig ( but no guarantie )
+
+IPv6
+==============
+
+- enable/disable it
+
+DHCP Client options
+====================
+
+- send back to dhcp server hostname
```

Comment:
I'd rephrase it e.g. like this: "Set hostname which is send back to the DHCP server"